admin/class-yoast-plugin-conflict.php

```diff
@@ -311,4 +319,15 @@ private function remove_deactivated_plugin() {
 			unset( $this->all_active_plugins[ $key_to_remove ] );
 		}
 	}
+
+	/**
+	 * Get the identifier from the plugin file
+	 *
+	 * @param string $plugin_file Plugin file to get Identifier from.
+	 *
+	 * @return string
+	 */
+	private function get_notification_identifier( $plugin_file ) {
+		return md5($plugin_file);
```

Comment:
Playing devil's advocate here: Why replace `sanitize_title` with `md5`? `sanitize_title` gives a more meaningful ID when debugging. Because `md5` is faster?